src/common/HPMSymbols.inc.h

```diff
@@ -309,6 +312,9 @@ HPExport const char *HPM_shared_symbols(int server_type)
 	if ((server_type&(SERVER_TYPE_MAP)) != 0 && !HPM_SYMBOL("chrif", chrif))
 		return "chrif";
 #endif // MAP_CHRIF_H
+#ifdef MAP_CLAN_H /* clan */
+if ((server_type&(SERVER_TYPE_MAP)) && !HPM_SYMBOL("clan", clan)) return "clan";
```

Comment:
please regenerate HPM auto generated code, because this look like outdated or added by hands